db/migrate/20180503074809_create_join_table_collection_topic.rb

```diff
@@ -0,0 +1,8 @@
+class CreateJoinTableCollectionTopic < ActiveRecord::Migration[5.1]
+  def change
+    create_join_table :collections, :topics do |t|
+      # t.index [:collection_id, :topic_id]
+      # t.index [:topic_id, :collection_id]
```

Comment:
Why are these indexes commented out?
Indexes help the database find things more quickly.